.eslintrc.cjs

```diff
@@ -51,6 +51,14 @@ module.exports = {
     'no-trailing-spaces'                       : ['error'],
     '@typescript-eslint/no-non-null-assertion' : 'off',
     '@typescript-eslint/ban-ts-comment'        : 'off',
-    'mocha/no-exclusive-tests'                 : 'warn'
-  }
+    'mocha/no-exclusive-tests'                 : 'warn',
+  },
+  overrides: [
```

Comment:
I think there is a point to allow var requires only in the .cjs files. Let's revert this part if you agree.

Reply:
that's what this does: allow var requires in CJS files

Reply:
Does it not also allow var requires in non-cjs files?

Reply:
the changes here: `if cjs file allow requires`

Reply:
I am dumb, thanks for pointing out the obvious...